bash-debugger/bdb.sh

```diff
@@ -1,36 +1,75 @@
 #!/bin/bash
+# shellcheck disable=SC2162,SC1090
+
+_e() { echo $'\e['"${1:-}"m ; }
+
+__dbg__usage() {
+    echo "bdb <command> [<command_args>...]"
+    echo
+    echo "Interactive bash script debugger."
+    echo "Runs the <command> with <command_args> and can trace execution steps and/or pause at set breakpoints."
+    echo
+    echo "Commands:"
+    __dbg_commands
+    echo
+}
+__dbg_commands() {
```

Comment:
Please use 2 underscores after `dbg` so that name will be consistent with others.